### DISCUSSION TITLE
FileNotFoundError: [Errno 2] No such file or directory:

### DISCUSSION THREAD

**Discussion by omkarade** [coqui-ai/TTS#1731] [General Q&A]
FileNotFoundError: [Errno 2] No such file or directory: '/content/drive/MyDrive/MainTTS/lock/LJSpeech/wavs/"ne0009152.wav'

after ruining !python3 /content/TTS/recipes/ttrain/glowtr/train_glowtts.py I got error .my file path is correct. Please help me.

**Comment by p0p4k** [comment]
There is a ' " ' in your filename. Might need to fix your transcript.txt file.

**Reply by omkarade** [reply]
how fix transcript.txt file.

**Reply by p0p4k** [reply]
Step 1. Open the transcript.txt file.
Step 2. Find " in the file and replace with blank ('').
Step 3. ???

**Reply by omkarade** [reply]
where transcript.txt file can found.

**Reply by p0p4k** [reply]
Metadata.csv or something like that. What does your dataset look like?
